operations/node_management/Update_the_HPE_Node_BIOS_Time.md

```diff
@@ -0,0 +1,71 @@
+# Update the HPE Node BIOS Time
+
+Check and set the time for HPE nodes.
+
+If the console log indicates the time between the rest of the system and the node is off by several hours, it prevents the spire-agent from getting a valid certificate, causing the node boot to drop into the dracut emergency shell.
+
+## Procedure
+
+1. Log in to a second terminal session in order to watch the node's console.
+
+    ***Please open this link in a new tab or page*** [Log in to a Node Using ConMan](../conman/Log_in_to_a_Node_Using_ConMan.md)
+
+    The first terminal session will be needed to run the commands to boot the node into the BIOS menu.
+
+1. (`ncn#`) In another terminal session set the following environment variables:
+
+    1. The BMC hostname.
+
+        ```bash
+        BMC=x3000c0s3b0
+        ```
+
+    1. Export the `root` user password of the BMC.
+
+        > NOTE: `read -s` is used to prevent the password from echoing to the screen or
+        > being saved in the shell history.
+
+        ```bash
+        read -r -s -p "${BMC} root password: " IPMI_PASSWORD
+        export IPMI_PASSWORD
+        ```
+
+1. (`ncn#`) Boot the node into its BIOS via the serial console.
+
+    When `ipmitool` is used to boot a HPE node into its BIOS menu it will not be available on on the nodes serial console, due to the node booting into a graphical BIOS menu. To access the serial version of the BIOS setup. Perform the `ipmitool` steps
```

Comment:
```suggestion
    When `ipmitool` is used to boot a HPE node into its BIOS menu, it will not be available on the node's serial console, because of the node booting into a graphical BIOS menu. In order to access the serial version of the BIOS setup, perform the `ipmitool` steps
```

Reply:
https://github.com/Cray-HPE/docs-csm/commit/ab3c45572fa5b86e6e61bbd196c6734919b6dae8#diff-77a8a9c40e37b9227e26cafc5bf523b5841d0e705bbd5c5c143b8ed99d87e17aR35